### ISSUE TITLE
Not working with `ListItem.Swipeable` from `react-native-elements`

### ISSUE THREAD

**Issue by KrisLau** [computerjazz/react-native-draggable-flatlist#382]
**Describe the bug**
Whenever I swap out my render item from `SwipeableRow` component (created using react-native-gesture-handler) with the `ListItem.Swipeable` from `react-native-elements`, the drag functionality stops working.

**To Reproduce**
https://snack.expo.dev/@kris_lau/draggable-flatlist-examples (Click on Swipeable)

**Platform & Dependencies**
Please list any  applicable dependencies in addition to those below (react-navigation etc).
 - react-native-draggable-flatlist version: 2.6.2
 - Platform: Android & iOS
 - React Native ~or Expo version~: 0.64.3
 - Reanimated version: 2.8.0
 - React Native Gesture Handler version: 1.10.3

**Additional context**
Sometimes the second or third item is draggable but the first one if pretty much always unresponsive if I use `ListItem.Swipeable`

**Comment by computerjazz** [comment]
Yeah, some gesture orchestration needs to happen between the swipe and drag gesture handlers.

I created https://github.com/computerjazz/react-native-swipeable-item to make sure swipe and drag could work together.
See the "swipeable" example here:  https://snack.expo.dev/@computerjazz/draggable-flatlist-examples

**Comment by KrisLau** [comment]
@computerjazz ahh ok i saw the component but i wanted to keep the app as small as possible so i wanted to use the libraries i already use. I'll just keep my current workaround where the user has to enable drag functionality where i swap out the parent container to just a plain `ListItem`
